jest.config.js

```diff
@@ -0,0 +1,195 @@
+/* eslint-disable max-len */
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // Respect "browser" field in package.json when resolving modules
+  // browser: false,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "/private/var/folders/7f/vrhn6c552bsghd95nn6m7pnc0000gn/T/jest_dx",
+
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // Indicates whether the coverage information should be collected while executing the test
+  // collectCoverage: false,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  collectCoverageFrom: [
+    '**/*.{js,jsx}',
+    '!**/node_modules/**',
```

Comment:
@aku2nnya I believe that this currently tells it to look within your node modules folder to calculate what % of your files you are testing. I'd recommend limiting this to your components/src folder!

Reply:
The "!" in the beginning actually states to not include it in the coverage (at least to my understanding), so it's being ignored :)